### PR TITLE
Missing elements

### PR DESCRIPTION
**Pull request by proghead00**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
Yes, I have.

### Description

(Write your answer here.)
This program will find out the missing elements in a series.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
I will be opening more PRs as each program I add.

### CODE REVIEW

File: C/cp/missingElementsFasterMethod.c [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
```
Comment (atarax665) [inline C/cp/missingElementsFasterMethod.c:1]: Add a beginner introduction of the problem and your approach.

File: C/cp/missingElementsFasterMethod.c [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#include <math.h>
+
+int main()
+{
+
+    int arr[6];
```
Comment (atarax665) [inline C/cp/missingElementsFasterMethod.c:7]: Why have you initialized the size of the array? You can take the size of the array as an input from the user.

File: C/cp/missingElementsFasterMethod.c [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        hrr[arr[i]]++;
+    }
+
+    printf("Missing elements: \n");
+    for (i = l; i <= h; i++)
+    {
+        if (hrr[i] == 0)
+            printf("\n %d", i);
+    }
+}
```
Comment (atarax665) [inline C/cp/missingElementsFasterMethod.c:33]: Add sample input/output and space and time complexity of the program.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by HarshCasper** [comment]
Closing the PR cause of no updates. Let me know if you would like this to be reopened.

**Pull request closed without merging**